### DISCUSSION TITLE
Issue encountered when installing aiortc from source

### DISCUSSION THREAD

**Discussion by htliu6** [aiortc/aiortc#1101] [Q&A]
# Issue Description
Hello everyone! I encountered some issues while attempting to install aiortc from the source code. After running `python3 setup.py install`, the console output included the following messages:

> /usr/lib/python3.8/distutils/dist.py:274: UserWarning: Unknown distribution option: 'cffi_modules'
>   warnings.warn(msg)
> running install
> running bdist_egg
> running egg_info
> writing UNKNOWN.egg-info/PKG-INFO
> writing dependency_links to UNKNOWN.egg-info/dependency_links.txt
> writing top-level names to UNKNOWN.egg-info/top_level.txt
> reading manifest file 'UNKNOWN.egg-info/SOURCES.txt'
> reading manifest template 'MANIFEST.in'
> writing manifest file 'UNKNOWN.egg-info/SOURCES.txt'
> installing library code to build/bdist.linux-x86_64/egg
> running install_lib
> warning: install_lib: 'build/lib' does not exist -- no Python modules to install
> 
> creating build/bdist.linux-x86_64/egg
> creating build/bdist.linux-x86_64/egg/EGG-INFO
> copying UNKNOWN.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO
> copying UNKNOWN.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
> copying UNKNOWN.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
> copying UNKNOWN.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
> zip_safe flag not set; analyzing archive contents...
> creating 'dist/UNKNOWN-0.0.0-py3.8.egg' and adding 'build/bdist.linux-x86_64/egg' to it
> removing 'build/bdist.linux-x86_64/egg' (and everything under it)
> Processing UNKNOWN-0.0.0-py3.8.egg
> Removing /usr/local/lib/python3.8/dist-packages/UNKNOWN-0.0.0-py3.8.egg
> Copying UNKNOWN-0.0.0-py3.8.egg to /usr/local/lib/python3.8/dist-packages
> UNKNOWN 0.0.0 is already the active version in easy-install.pth
> 
> Installed /usr/local/lib/python3.8/dist-packages/UNKNOWN-0.0.0-py3.8.egg
> Processing dependencies for UNKNOWN==0.0.0
> Finished processing dependencies for UNKNOWN==0.0.0

Subsequently, when I tried to run the server example using `python3 server.py`, I encountered the following error:

> Traceback (most recent call last):
> File "server.py", line 11, in <module>
> from aiortc import MediaStreamTrack, RTCPeerConnection, RTCSessionDescription
> ModuleNotFoundError: No module named 'aiortc'

# Relevant Information
- Operating System: Ubuntu 20.04
- Python Version: 3.8.10
- aiortc Version: 1.8.0

Has anyone else faced a similar situation, or does anyone know how to resolve this problem? Thanks!

**Comment by hilookas** [comment]
try: pip install --upgrade pip
